sci-biology/salmon/salmon-0.6.0.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
Another aspect I discussed with submitter earlier is that he works a lot from prefix and his prefix doesn't support EAPI 6 yet - I don't know if support is currently lacking in prefix or if submitter is a lazy upgrader. Prefix has been in flux for the past few weeks so I wouldn't be surprised if EAPI 6 was messy over there.